packages/contracts/contracts/test/UpgradeGovernor.t.sol

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { UpgradeGovernor_Initializer } from "./CommonTest.t.sol";
+import { Types } from "../libraries/Types.sol";
+import { Proxy } from "../universal/Proxy.sol";
+
+contract UpgradeGovernorTest is UpgradeGovernor_Initializer {
+    uint256 blockNumber = 0;
+    enum ProposalState {
+        Pending,
+        Active,
+        Canceled,
+        Defeated,
+        Succeeded,
+        Queued,
+        Expired,
+        Executed
+    }
+    function setUp() public virtual override {
+        super.setUp();
+        _roll(1);
+        // minting to guardians
+        vm.prank(owner);
+        securityCouncilToken.safeMint(guardian1, baseUri);
+        vm.prank(owner);
+        securityCouncilToken.safeMint(guardian2, baseUri);
+
+        vm.prank(guardian1);
+        securityCouncilToken.delegate(guardian1);
```

Comment:
IMHO, there is no need to delegate token to guardian1 since the token is already minted to guardian1.
Why is this required?